docs/tutorials/python/trip-booking-app/code/workflows.py

```diff
@@ -55,7 +55,6 @@ async def run(self, book_input: BookVacationInput):
                 book_hotel,
                 book_input,
                 start_to_close_timeout=timedelta(seconds=10),
-                maximum_attempts=book_input.attempts,
```

Comment:
In this case, the mistake is that `maximum_attempts` is defined in the wrong place (outside of the RetryPolicy instead of inside it). In the interest of time, I will accept this PR as-is and then add it back in the correct place through another PR.